conf/nginx.conf

```diff
@@ -32,12 +32,10 @@ http {
   }
 
   # 1m stands for 1 megabyte so the zone can store ~8k users.
-  # User's typically don't go over 1rps including bots so set the
-  # generic rate limit of all endpoints to 1rps.
-  limit_req_zone $limit_per_user zone=badge_user_1rps_limit:1m rate=1r/s;
-  limit_req_zone $limit_per_user zone=assets_user_1rps_limit:1m rate=1r/s;
-  limit_req_zone $limit_per_user zone=create_ann_user_1rps_limit:1m rate=1r/s;
-  limit_req_zone $limit_per_user zone=user_1rps_limit:1m rate=1r/s;
+  limit_req_zone $limit_per_user zone=badge_user_limit:1m rate=1r/s;
+  limit_req_zone $limit_per_user zone=assets_user_limit:1m rate=20r/s;
+  limit_req_zone $limit_per_user zone=create_ann_user_limit:1m rate=4r/s;
+  limit_req_zone $limit_per_user zone=user_limit:1m rate=4r/s;
   limit_req_status 429;
```

Comment:
nb. `badge_user_limit` has been left at a low value since it is low-value compared to other requests, and the latency is acceptable in the browser extension's UX.